lib/jekyll/assets/plugins/proxy/optim.rb

```diff
@@ -25,6 +25,7 @@ def initialize
 
         def process
           optimc = @env.asset_config[:plugins][:img][:optim]
+          return @file if optimc == false
```

Comment:
I think I'd be okay with just a `return @file if !optimc` that way nil disables it too. I never really intended it to be the default, in any circumstance so nil should really disable it too no? But we have a bigger problem if it's running by default.

Reply:
Updated